common/network/IPV4Address.cpp

```diff
@@ -93,7 +95,17 @@ bool IPV4Address::IsWildcard() const {
 string IPV4Address::ToString() const {
   struct in_addr addr;
   addr.s_addr = m_address;
+#ifdef HAVE_INET_NTOP
+  char str[INET_ADDRSTRLEN];
+  if (inet_ntop(AF_INET, &addr, str, INET_ADDRSTRLEN) == NULL) {
+    OLA_WARN << "Failed to convert address to string using inet_ntop, failing "
```

Comment:
We can't return false, we could return the empty string, but that's confusing. If somehow "ENOSPC The converted address string would exceed the size given by size." then inet_ntoa would still succeed.